### DISCUSSION TITLE
Changing font size in Beamer presentation?

### DISCUSSION THREAD

**Discussion by maxdrohde** [quarto-dev/quarto-cli#5025] [Q&A] [answered]
Am I changing the font size the correct way? I ran this test file, and am seeing no change when I change the fontsize argument. I also tried it with and without the "pt".

````

---
title: "Beamer font test"
format:
  beamer:
    fontsize: "25pt"
---

This is a test.

````

**Comment by cderv** [comment, marked as answer]
I believe this is working 

`fontsize` variable will be put in `\document[24pt]{beamer}` and this is the expected way. See this doc: https://latex-beamer.com/tutorials/beamer-font/#Font-size

Taking their example, try this doc: 

````markdown
---
title: "Beamer font test"
format:
  beamer:
    fontsize: "24pt"
keep-tex: true
---

## Font size in Beamer (default)

\tiny This is tiny font size  
\scriptsize This is scriptsize font size  
\footnotesize This is footnotesize font size  
\small This is small font size  
\normalsize This is normalsize font size  
\large This is large font size  
\Large This is Large font size  
\LARGE This is LARGE font size  
\huge This is huge font size  
\Huge This is Huge font size  
````

This is default without `fontsize`
![image](https://user-images.githubusercontent.com/6791940/228675545-724ffdea-2015-4e3d-8e0e-284304734a2e.png)

This is with `fontsize: 8pt`
![image](https://user-images.githubusercontent.com/6791940/228675812-0a4792d2-7304-4cf2-8c8a-dc74a0e16bb8.png)


I am not sure why some value like 24pt does not have effect 🤔 
Seems like a LaTeX thing, maybe with a limite based on space on slide ? 

Documentclass in `.tex` file correctly change to 
````
\documentclass[
  24pt,
  ignorenonframetext,
]{beamer}
````

so the fontsize setting get passed. 

Maybe this is because Beamer only support a default set of fontsize. I get that from https://tex.stackexchange.com/a/314507/209358 which points to the manual https://tug.ctan.org/macros/latex/contrib/beamer/doc/beameruserguide.pdf
In part 
> 18.2.1 Choosing a Font Size for Normal Text
they list the different value possible and 24pt is not an option 
![image](https://user-images.githubusercontent.com/6791940/228677259-d32b08d3-be68-4568-9229-f3f1d1b946a4.png)


Try with 20pt and it should work for the biggest you can get. Read the manual further about fonts size to understand the why behind this in Beamer. 

Hope it helps

**Reply by maxdrohde** [reply]
Thank you!
